.drone.yml

```diff
@@ -91,14 +133,33 @@ steps:
   image: grafana/build-container:1.4.1
   commands:
   - ./bin/grabpl build-plugins --jobs 8 --edition oss --no-install-deps
+  when:
+    paths:
+    - go.mod
+    - go.sum
+    - pkg/**
+    - package.json
+    - yarn.lock
+    - public/**
+    - packages/**
+    - packaging/docker/**
   depends_on:
   - initialize
-  - lint-backend
 
 - name: gen-version
   image: grafana/build-container:1.4.1
   commands:
   - ./bin/grabpl gen-version --build-id ${DRONE_BUILD_NUMBER}
+  when:
+    paths:
+    - go.mod
+    - go.sum
```

Comment:
not sure here about the script gen-version :) need to take a look, but if it is for new version, i think cue/** should be in the condition.